packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -2274,6 +2285,25 @@ class WebpackCLI implements IWebpackCLI {
       return options;
     };
 
+    if (!!options.dotEnv && !this.checkPackageExists("webpack")) {
```

Comment:
We already check the webpack existence when running the `webpack-cli build` command.

Reply:
The smoke test "webpack missing" fails without this check

Reply:
Can you share a corresponding ci run where it failed? I'll look at the error once.

Reply:
https://github.com/webpack/webpack-cli/actions/runs/4879057741/jobs/8705268337